modules/system/console/WinterInstall.php

```diff
@@ -442,6 +456,17 @@ protected function writeToConfig($file, $values)
         $this->configWriter->toFile($configFile, $values);
     }
 
+    protected function writeToEnv($values)
+    {
+        $path = base_path('.env');
+
+        foreach ($values as $key => $value) {
+            $this->envFileWriter->set($key, $value);
```

Comment:
This should probably instead happen in the writeToConfig method with a mapping of file / key values to env key values and a check to see if the .env file exists to decide if values are written to `.env` or if they're written to the config files. If the .env file exists, then values should only be written there, not in both locations.

@bennothommo @jaxwilko any thoughts on that?

Reply:
I don't necessarily mind it going in both locations as long as the `.env` file acts as an override (which it should given we have `env()` calls in the config). It would at least allow people who don't want to use `.env` files to still be able to install and use Winter.

Reply:
My point with that is to avoid writing config values into source control unintentionally.